Objects.md

```diff
@@ -4,28 +4,43 @@ Objects can be found throughout the levels in SuperTux.
+![alt text](https://github.com/SuperTux/data/blob/master/images/objects/switch/switch-0.png?raw=true "Switch")
+
+Switches can either be levers or buttons. Levers work by pressing the up key and buttons by jumping on them. They both run scripts, which can change the level or do something to Tux. In most cases, it will add a path to make part of the level possible. Sometimes there will be two signs of the same colour at the switch and the thing that gets changed by the script. 
 
 ### Platforms
 
-Platforms take Tux to and from places. Some are always on, some move when Tux gets on them and others need to be started by a switch or a script. ![alt text](https://github.com/SuperTux/data/blob/master/images/objects/flying_platform/flying_platform-0.png?raw=true "Platform")
+![alt text](https://github.com/SuperTux/data/blob/master/images/objects/flying_platform/flying_platform-0.png?raw=true "Platform")
+
+Platforms take Tux to and from places. Some are always on, some move when Tux gets on them and others need to be started by a switch or a script. 
 
 ### Info Blocks
 
-Info Blocks can give valuable information about the level ahead. They can be hit from below or butt-jumped from on top. ![alt text](https://github.com/SuperTux/data/blob/master/images/objects/bonus_block/infoblock.png?raw=true "Info Block")
+![alt text](https://github.com/SuperTux/data/blob/master/images/objects/bonus_block/infoblock.png?raw=true "Info Block")
+
+Info Blocks can give valuable information about the level ahead. They can be hit from below or butt-jumped from on top. 
 
 ### Trampolines
 
-Trampolines are available fixed and portable. They can be jumped on to help Tux reach new heights. Portable ones can be moved with the action key. There are also ones that rust and break after a few jumps. ![alt text](https://github.com/SuperTux/data/blob/master/images/objects/trampoline/trampoline1-0.png?raw=true "Portable Trampoline")![alt text](https://github.com/SuperTux/data/blob/master/images/objects/trampoline/trampoline2-0.png?raw=true "Fixed Trampoline")
+![alt text](https://github.com/SuperTux/data/blob/master/images/objects/trampoline/trampoline1-0.png?raw=true "Portable Trampoline")
+![alt text](https://github.com/SuperTux/data/blob/master/images/objects/trampoline/trampoline2-0.png?raw=true "Fixed Trampoline")
+
+Trampolines are available fixed and portable. They can be jumped on to help Tux reach new heights. Portable ones can be moved with the action key. There are also ones that rust and break after a few jumps.
```

Comment:
Show which is which (fixed, portable). Also include something about Lanterns... Change "moved" to "carried" or "picked up"